contracts/Loans.vy

```diff
@@ -650,6 +648,7 @@ def reserveEth(
     _interest: uint256,
     _maturity: uint256,
     _collaterals: DynArray[Collateral, 100],
+    _delegations: DynArray[bool, 100],
```

Comment:
Here we're enabling support for different delegation wallets, one per collateral, right? I understand this may be important if we want to enable per-NFT control but to also allow for more gas-efficient calls, we should also create a variable that states "delegate all NFTs to the borrower's wallet". What do you think?

Reply:
As discussed, simplifying by having a single bool parameter to either set up delegates for all collaterals or none.